### PR TITLE
vote fix. now will round to higher difficulty.

### PR DESCRIPTION
**Pull request by maksiu1000**
### Brief description of the changes:
---old. if vote were 1,1,3,3  difficulty is set to 1. after fix it will set 3---

### Tested Changes
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

**Review by kajendra — COMMENTED** [review]
This code will work.
I believe patrol is working on a voting system that does two rounds.
Similar to run-off ballot.
[Refactor Voting](https://github.com/AwesomePatrol/ComfyFactorio/tree/refactor-voting)

File: maps/biter_battles_v2/difficulty_vote.lua [hunk 1]
```diff
@@ -71,7 +71,7 @@ local function set_difficulty()
 		vote_count = vote_count + 1
 	end
 	if vote_count == 0 then return end
-	v= math.round(vote_count/2, 0)
+	v= math.floor(vote_count/2, 0)+1
```
Comment (guymaor86) [inline maps/biter_battles_v2/difficulty_vote.lua:74]: i would say do `math.ceil` instead of doing of floor+1

Comment (guymaor86) [reply maps/biter_battles_v2/difficulty_vote.lua:74]: also floor/ceil only take one argument as opposed to `round` that take two

Comment (maksiu1000) [reply maps/biter_battles_v2/difficulty_vote.lua]: if votes are (easy, easy, hard, hard) 
with floor +1 it goes like this. 4/2 + 1 = 3 = hard.
with ceil it is. 4/2 = 2 = easy.
whatever if patrol going make brand new one :D

Comment (guymaor86) [reply maps/biter_battles_v2/difficulty_vote.lua]: but for the case you noted, no rounding is necessary
was this decided on this discord? maybe i missed it

Comment (maksiu1000) [reply maps/biter_battles_v2/difficulty_vote.lua]: what if there is 5 votes.
 it wasn't decided on discord. i wrote that median thing 1-2 month ago and i just realized it's not exactly what i planned :D imo its so minor its not worth discussion.

**Pull request merged**